srcpkgs/wired-notify/patches/32bit.patch

```diff
@@ -0,0 +1,18 @@
+--- a/src/manager.rs
++++ b/src/manager.rs
+@@ -143,6 +143,6 @@
+             if let Some(threshold) = Config::get().idle_threshold {
+                 match maths_utility::query_screensaver_info(&self.base_window) {
+                     Ok(info) => {
+-                        if info.idle / 1000 >= threshold {
++                        if info.idle as u64 / 1000 >= threshold {
+                             self.layout_windows
+                                 .values_mut()
+                                 .flatten()
+				 .for_each(|w| w.update_mode = UpdateModes::DRAW);
+                        }
+                    },
+                    Err(e) => eprintln!("{}", e),
+                }
+            }
+        }	
```

Comment:
```suggestion
```
I don't think this section is necessary for patching properly

Reply:
In truth, even in my opinion it is not necessary ..
The only reason I inserted those extra lines is because during the patch phase I was getting two errors (both `patch unexpectedly ends in middle of line`).
By inserting those lines I should have deleted one, although the other still remains.
If you think I can delete them, I change them immediately, also because without all those superfluous lines the patch is much cleaner.

Reply:
hmm *removing* these lines removed that warning for me

Reply:
Ah, wow, I remove them instantly!

Reply:
ok, I think I know what's going on. your editor isn't adding a newline at the end of the file, so it's not the exact same as what's in the file its patching. `echo "" >> srcpkgs/wired-notify/patches/32bit.patch` should fix it now I think

Reply:
Oh, I get it.
So as soon as the check builds finish I add the newline.